orange/src/test/java/com/hartwig/hmftools/orange/algo/purple/ChromosomalRearrangementsDeterminerTest.java

```diff
@@ -0,0 +1,103 @@
+package com.hartwig.hmftools.orange.algo.purple;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.common.collect.Lists;
+import com.hartwig.hmftools.common.genome.refgenome.RefGenomeCoordinates;
+import com.hartwig.hmftools.common.purple.PurpleCopyNumber;
+import com.hartwig.hmftools.common.purple.PurpleTestUtils;
+import com.hartwig.hmftools.datamodel.orange.OrangeRefGenomeVersion;
+
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+
+public class ChromosomalRearrangementsDeterminerTest
+{
+    private static final ChromosomalRearrangementsDeterminer chromosomalRearrangementsDeterminer =
```

Comment:
private static final variables should be named like `CHROMOSOMAL_REARRANGEMENTS_DETERMINER`